index.js

```diff
@@ -72,3 +71,15 @@ function makeHash (markdownString, opts) {
 
   return hasha(markdownString + optsString)
 }
+
+function createProcessor (before) {
+  return unified()
+    .use(markdown)
+    .use(before)
+    .use(emoji)
+    .use(remark2rehype)
+    .use(slug)
+    .use(autolinkHeadings, { behavior: 'wrap' })
+    .use(highlight)
+    .use(html)
```

Comment:
Switched this up to use unified/remark/rehype directly, it’s a bit more explicit and needs less dependencies in memory